### DISCUSSION TITLE
Ansible UI integration

### DISCUSSION THREAD

**Discussion by arminus** [SquirrelCorporation/SquirrelServersManager#37] [Ideas]
I'm kind of comparing this to sempahore... Generally, it feels to me like the Ansible UI integration is a bit "hidden", this starts with the Playbooks page.

Typically, I run playbooks for a set of boxes, so my natural entry point would have been the playbooks page. But I can't execute playbooks from there at all. Being able to do an apply to all from the devices page is fine, but this is maybe not where one would intuitively look first.

Also, it would be nice if devices could be grouped into inventories so that playbooks could be run for a subset of devices only.

Just some thoughts based on how I use semaphore...

**Comment by chopeta** [comment]
Here's different ways you can execute a playbook:
https://squirrelserversmanager.io/docs/exec-playbook

What I do miss is the ability to add a playbook execution to the crontab which seems to be already part of the roadmap.

**Comment by SquirrelDeveloper** [comment]
Hi, thanks for your contribution, will take a look at possible implementation soon

**Comment by SquirrelDeveloper** [comment]
@arminus A new release has been published, and your others comments have been added to the backlog
